RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -393,4 +403,8 @@ extension NonSubscriptionTransaction: Transaction {
     var isCancelled: Bool {
         false
     }
+
+    var subscriptionManagementURL: URL? {
```

Comment:
should this be `managementURL`? `subscriptionManagementURL` doesn't make much sense for a non-subscription

Reply:
Yeah this was part of a test, but forgot to revert it. On it 👍